### ISSUE TITLE
let user (somehow) read about scenarios before selecting from dropdown

### ISSUE THREAD

**Issue by yongrenjie** [alan-turing-institute/demoland-web#39]
@ciupava's suggestion, which i think makes lots of sense (because the dropdown text doesn't tell you anything about what you're selecting)

**Comment by martinfleis** [comment]
Link https://ciupava.github.io/LandUseDemonstrator/book/scenarios.html?

**Comment by ciupava** [comment]
Yeah, we could link to these descriptions from the book., not sure where from.
Right now one can only know about the scenarios when clicking on the dropdown menu, so wouldn't know what to select unless switches among them all, so it'd be helpful to have them all outlined together.

**Comment by yongrenjie** [comment]
I think I'm going to close this for now, because the original issue is sort of fixed in https://github.com/alan-turing-institute/demoland-web/commit/bc93edefa7e4eeb68e3aa41ce30d08b28a3e31e7 — there is a separate issue which Anna raised in that it's not 100% clear that the description relates to the main scenario (rather than the scenario being compared against), which I'll reopen as a separate issue
